### PR TITLE
Pharmaceuticals proofreading

### LINKED ISSUE #981: Manual References for Therapeutics (Pharmaceuticals)
Reference information is not rendering for the following citations:
- https://www.fda.gov/media/138946/download
- https://www.fda.gov/media/150319/download
- https://www.fda.gov/media/145801/download
- https://www.phe.gov/emergency/events/COVID19/investigation-MCM/Bamlanivimab-etesevimab/Pages/bamlanivimab-etesevimab-distribution-pause.aspx
- https://www.contractpharma.com/contents/view_breaking-news/2020-08-31/fda-clears-spis-ebselen-for-phase-ii-covid-19-trials

@agitter, apologies that I haven't caught onto this by now, but when you add the manual references, are you manually inserting metadata in the json file? I'm happy to add these but just want to make sure I follow the correct procedure!

### PR DESCRIPTION
**Pull request by agitter**
Closes #981

This is ready for review and discussion.  I used HTML comments for discussion points.  We don't necessary need to change any text in those sections.

Do we want to comment on Ivermectin?  It's a drug that has come up frequently in the (anti-vax) media that may benefit from an assessment of the current evidence.  Some of that is summarized [here](https://gidmk.medium.com/is-ivermectin-for-covid-19-based-on-fraudulent-research-5cc079278602) though there may be even more recent developments like this [trial](https://www.latimes.com/business/story/2021-08-11/ivermectin-no-effect-covid) that showed no effect.

We'll need to double-check the manual references work in the HTML artifact.

I haven't coordinated my edits with other concurrent edits (#996 #997).  I see there is a merge conflict.

### CODE REVIEW

**Comment by agitter** [comment]
I reviewed the references in the appendix and pushed some more manual updates.

File: content/20.pharmaceuticals.md [hunk 1]
```diff
@@ -421,19 +431,20 @@ After the design and confirmation of N3 as a highly potent Michael acceptor inhi
...
 While there is clear computational and _in vitro_ support for ebselen's potential as a COVID-19 therapeutic, results from clinical trials are not yet available for this compound.
-In July 2020 phase II clinical trials commenced to assess the effects of SPI-1005, an investigational drug from Sound Pharmaceuticals that contains ebselen [@url:https://www.contractpharma.com/contents/view_breaking-news/2020-08-31/fda-clears-spis-ebselen-for-phase-ii-covid-19-trials/], on 60 adults presenting with each of moderate [@clinicaltrials:NCT04484025] and severe [@clinicaltrials:NCT04483973] COVID-19.
+In July 2020 phase II clinical trials commenced to assess the effects of SPI-1005, an investigational drug from Sound Pharmaceuticals that contains ebselen [@url:https://www.contractpharma.com/contents/view_breaking-news/2020-08-31/fda-clears-spis-ebselen-for-phase-ii-covid-19-trials], on 60 adults presenting with each of moderate [@clinicaltrials:NCT04484025] and severe [@clinicaltrials:NCT04483973] COVID-19.
 
 Another candidate has emerged from a broader assessment of work in computational prediction of candidate drugs for repurposing against COVID-19.
```
Comment (cbrueffer) [inline content/20.pharmaceuticals.md:436]: Not part of your changes, but I found this sentence quite confusing. Candidate sounds like it refers to a drug candidate, but no specific one is mentioned in the paragraph. Maybe `Another candidate -> Other candidates`?

File: content/20.pharmaceuticals.md [hunk 2]
```diff
@@ -421,19 +431,20 @@ After the design and confirmation of N3 as a highly potent Michael acceptor inhi
...
-A second study expanded on these results by identifying that cationic amphiphilicity was a shared property among many of the candidate drugs identified through both computational and phenotypic screens, suggesting it as a potential cause of a compound's proposed antiviral activity [@doi:10.1101/2021.03.23.436648].
+A second study expanded on these results by identifying that cationic amphiphilicity was a shared property among many of the candidate drugs identified through both computational and phenotypic screens, suggesting it as a potential cause of a compound's proposed antiviral activity [@doi:10.1126/science.abi4708].
 Cationic amphiphilic drugs are known to initiate phospholipidosis, which is when phospholipids accumulate in the lysosome [@doi:10.1515/hsz-2019-0270].
 Mechanistically, phospholipidosis could disrupt viral replication by inhibiting lipid processing [@doi:10.1080/01913123.2020.1850966] (see discussion of HCQ in Appendix A).
 They found that antiviral activity against SARS-CoV-2 was correlated with phospholipidosis for drugs with and without cationic amphiphilicity.
```
Comment (rando2) [inline content/20.pharmaceuticals.md:443]:
```suggestion
They found that antiviral activity against SARS-CoV-2 _in vitro_ was correlated with phospholipidosis for drugs with and without cationic amphiphilicity.
```

File: content/20.pharmaceuticals.md [hunk 3]
```diff
@@ -421,19 +431,20 @@ After the design and confirmation of N3 as a highly potent Michael acceptor inhi
...
 This observation is also consistent with preliminary evidence that certain psychotropic drugs may be protective against COVID-19 [@doi:10.1016/j.drudis.2020.06.022; @doi:10.1007/s00406-020-01231-x; @doi:10.1001/jama.2020.22760].
```
Comment (rando2) [inline content/20.pharmaceuticals.md:444]:
```suggestion
This observation would be consistent with preliminary evidence that certain psychotropic drugs may be protective against COVID-19 [@doi:10.1016/j.drudis.2020.06.022; @doi:10.1007/s00406-020-01231-x; @doi:10.1001/jama.2020.22760].
```

File: content/20.pharmaceuticals.md [hunk 4]
```diff
@@ -421,19 +431,20 @@ After the design and confirmation of N3 as a highly potent Michael acceptor inhi
...
 While _in vivo_ application of this principle has not yet been successful, this effort demonstrates the potential for high-throughput screening to reveal general biological principles that may be beneficial in developing strategies to combat SARS-CoV-2.
```
Comment (rando2) [inline content/20.pharmaceuticals.md:445]:
```suggestion
However, these effects may not hold _in vivo_.
While clinical application of high-throughput screening for compounds against SARS-CoV-2, this effort demonstrates its potential to reveal general biological principles that may be beneficial in developing strategies to combat SARS-CoV-2.
```

File: content/20.pharmaceuticals.md [hunk 5]
```diff
@@ -421,19 +431,20 @@ After the design and confirmation of N3 as a highly potent Michael acceptor inhi
...
 Other large-scale efforts to screen candidate drugs against SARS-CoV-2 (e.g., [@doi:10.1016/j.celrep.2021.108959]) may complement these efforts and provide additional insights into the fundamental characteristics of drugs that show potential for repurposing.
+<!-- I read the Tummino et al. study, and this isn't how I woudl characterize their findings. My understanding is that they hypothesize that these cationic amphiphilic drugs induce phospholipidosis, which has antiviral activity in in vitro assays but none in vivo. They are characterizing these compounds as confounders or false positives that were detected in the computational or experimental screens and define chemical structure-based rules to identify similar false positives. The current framing seems to imply that cationic amphiphilic drugs may still have clinical success, but the paper argues that pursuing these confounders (including CQ/HCQ) has been a big waste of time and money in clinical trials. I read the journal version, not the preprint, so the message may have changed. -->
```
Comment (rando2) [inline content/20.pharmaceuticals.md:447]: Thank you so much for catching this! I read it a while ago and I don't remember the details at all. Would the comments above address your concerns?

File: content/20.pharmaceuticals.md [hunk 6]
```diff
@@ -456,6 +467,7 @@ Because the second phase of COVID-19 infection, associated with a severe inflamm
 Once again, this approach does not require the development of compounds specifically designed for the emergent viral threat, but rather hinges on a theoretical understanding of how the virus interacts with a human host and how pharmaceuticals can be used to modify those interactions.
 In contrast, other mAbs can be designed based on a specific pathogen.
 COVID-19 has demonstrated that this development can progress rapidly, with two mAb cocktails currently approved in the United States.
+<!-- Do we have any broad insights into the types of trials that have been run and the degree of clinical evidence they provide? It seems as if in several cases there were small initial trials (some observational or underpowered) that didn't provide strong evidence for or against a drug. We had to wait for RECOVERY, Solidarity, etc. results to draw better conclusions. Is that accurate or worth commenting on? -->
```
Comment (rando2) [inline content/20.pharmaceuticals.md:470]: I'll work on opening a PR to address this and the comment below!

**Review by rando2 — APPROVED** [review]

File: content/20.pharmaceuticals.md [hunk 7]
```diff
@@ -350,7 +357,7 @@ Etesevimab was shown to bind a different epitope on the spike protein than bamla
 To assess the efficacy and safety of bamlanivimab alone or in combination with etesevimab for the treatment of COVID-19, a phase 2/3 trial (BLAZE-1) [@clinicaltrials:NCT04427501] was initiated.
 The interim analysis of the phase 2 portion suggested that bamlanivimab alone was able to accelerate the reduction in viral load [@doi:10.1056/NEJMoa2029849].
 However, more recent data suggests that only the bamlanivimab/etesevimab combination therapy is able to reduce viral load in COVID-19 patients [@doi:10.1001/jama.2021.0202].
-Based on this data, the combination therapy received an EUA for COVID-19 from the FDA in February of 2021 [@url:https://www.fda.gov/media/145801/download].
+Based on this data, the combination therapy received an EUA for COVID-19 from the FDA in February 2021 [@url:https://www.fda.gov/media/145801/download].
```
[the remark this answers is not in the document]
Comment (agitter) [reply content/20.pharmaceuticals.md:360]: I had removed the "of".  We didn't use it with dates elsewhere.

File: content/20.pharmaceuticals.md (hunk 5 above)
Comment (agitter) [reply content/20.pharmaceuticals.md:447]: I think so.  I'll commit all your changes and then read through the paragraph again.

File: content/20.pharmaceuticals.md (hunk 6 above)
Comment (agitter) [reply content/20.pharmaceuticals.md:470]: Great!  We can leave these comments here for now and resolve them in a future PR after merging this one, which already touches many files.

File: content/20.pharmaceuticals.md (hunk 5 above)
Comment (agitter) [reply content/20.pharmaceuticals.md]: I made additional changes in 3dcd7e2a93d9c4bc0ab2078d5d664fa0325900b8 to match my interpretation of Tummino et al.  How does that look to you?

File: content/20.pharmaceuticals.md (hunk 6 above)
Comment (rando2) [reply content/20.pharmaceuticals.md:466]: The more I think about this comment, the more I think this is really the crux of the paper.

**Pull request merged**